backend/src/executor.rs

```diff
@@ -428,7 +429,14 @@ impl ExecutorConfig {
                 dirs::home_dir().map(|home| home.join(".gemini").join("settings.json"))
             }
             ExecutorConfig::SstOpencode => {
-                xdg::BaseDirectories::with_prefix("opencode").get_config_file("opencode.json")
+                #[cfg(unix)]
+                {
+                    xdg::BaseDirectories::with_prefix("opencode").get_config_file("opencode.json")
```

Comment:
Why do we need this library in this case? Why can't we do `dirs::config_dir().join(...)`?

Reply:
`sst opencode` uses XDG on macOS. `dirs::config_dir` returns `~/Library/Application Support/...` instead of `~/.config`